shell/components/demo/firebase-upload-demo.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://www.gstatic.com/firebasejs/4.2.0/firebase.js"></script>
+<script type="module" src="../../app-shell/firebase-config.js"></script>
+<script type="module" src="../firebase-upload.js"></script>
+
+<firebase-upload><span>Click-to-Upload</span></firebase-upload>
```

Comment:
No demo for multiple, maybe intentional since it's not fully implemented yet (see my comment below) but could be worth an html comment in here to note we should add when we do support it.